cli/internal/publish/plugins.go

```diff
@@ -26,10 +43,115 @@ type PackageJSONV1 struct {
 }
 
 type TargetBuild struct {
-	OS       string `json:"os"`
-	Arch     string `json:"arch"`
-	Path     string `json:"path"`
-	Checksum string `json:"checksum"`
+	OS             string `json:"os"`
+	Arch           string `json:"arch"`
+	Path           string `json:"path"`
+	Checksum       string `json:"checksum"`
+	DockerImageTag string `json:"docker_image_tag"`
+}
+
+type LoadResponse struct {
+	Stream string `json:"stream"`
+}
+
+type dockerProgressReader struct {
+	decoder         *json.Decoder
+	bar             *progressbar.ProgressBar
+	layerPushedByID map[string]int64
+	totalBytes      int64
+}
+
+type dockerProgressInfo struct {
+	Status       string `json:"status"`
+	Progress     string `json:"progress"`
+	ProgressData struct {
+		Current int64 `json:"current"`
+		Total   int64 `json:"total"`
+	} `json:"progressDetail"`
+	LayerID     string `json:"id"`
+	ErrorDetail struct {
+		Message string `json:"message"`
+	} `json:"errorDetail"`
+}
+
+type transportWithRegistryAuth struct {
+	http.RoundTripper
+	baseTransport *http.Transport
+	registryAuth  string
+}
+
+func newTransportWithRegistryAuth(insecureSkipVerify bool, registryAuth string) *transportWithRegistryAuth {
+	baseTransport := http.DefaultTransport.(*http.Transport).Clone()
+	baseTransport.TLSClientConfig.InsecureSkipVerify = insecureSkipVerify
```

Comment:
I think we want to remove this for prod, right?

Reply:
Only done for `localhost` images https://github.com/erezrokah/cloudquery/blob/6086d56a7a4f9d07bfc2e8ec2684776cad1537cb/cli/internal/publish/plugins.go#L598